### DISCUSSION TITLE
send message from react to unity with bool parameter

### DISCUSSION THREAD

**Discussion by tjumma** [jeffreylanters/react-unity-webgl#191] [Help Wanted!] [answered]
Hello, sir! I'm trying to send a message from React to Unity with a bool parameter, but am getting this error 

`unityContext.send("ReactToUnity", "SetWalletConnected", true);`

![DApp Starter _ Solana - Brave 2021-05-22 23 24 42](https://user-images.githubusercontent.com/55762319/119239935-450c9980-bb55-11eb-96c2-a4341a009cd6.png)

Sending messages with int parameter works fine. Could you help me figure out what I am doing wrong?

**Comment by jeffreylanters** [comment, marked as answer]
Hi! This is a limitation from Unity's side, a Boolean value is not supported as of the latest Unity version. There are a couple ways you could work around this, but I think the easiest and most pretty way would be to either send some JSON data, or just send an integer instead. I hope this helps you, good luck with your project!

**Reply by tjumma** [reply]
Thanx a lot for such a quick response! It definitely helped
